packages/static-wado-creator/lib/dictionary/Tags.js

```diff
@@ -1,5 +1,33 @@
 const dataDictionary = require("./dataDictionary");
 
+/** Find the actual tag for a private value */
+const findPrivate = (item, tagObject, create) => {
```

Comment:
This is the bulk of the work - just return the actual tag to use via looking it up with a search (ugly, but works)

Reply:
private tags are inherently not pretty, but this looks fine - I'm wondering why you stopped at 0x3F (comment says 0x30) - performance?